aws/resource_aws_backup_region_settings_test.go

```diff
@@ -0,0 +1,123 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/backup"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccAwsBackupRegionSettings_basic(t *testing.T) {
+	var settings backup.DescribeRegionSettingsOutput
```

Comment:
This test is failing in GovCloud as FSx is not available there, can we use [describeRegionSettings](https://docs.aws.amazon.com/aws-backup/latest/devguide/API_DescribeRegionSettings.html) to populate the list to check? cc: @YakDriver

Reply:
The api allows for partial update (but describe actually return all services) so I can add computed to the map to reduce service keys in tests to something simple like RDS. I just wanted it to prevent any state drift issues

Reply:
See below